constants.py

```diff
@@ -3,10 +3,12 @@
 import sys
 import random
 
+DISPLAY_SIZE = (800, 600)
+
 MAIN_DIR = os.path.dirname(__file__)
 IMAGES_DIR = os.path.join(MAIN_DIR, "resources/images")
 
-DISPLAY_SIZE = (800, 600)
+TARGET_IMAGE = os.path.join(IMAGES_DIR, "target.png")
 
```

Comment:
You are great at using constants, I definitely need to get better at this. These variables clean up our code and make it more readable. I'm not sure if having constants in their own file is common practice but I'm okay with it.